eslint.config.js

```
@@ -34,14 +34,15 @@ export default tseslint.config(
     },
     rules: {
       // Turn all rules to warnings instead of errors
-      "no-unused-vars": "warn",
+      "no-unused-vars": ["warn", { "argsIgnorePattern": "^_", "varsIgnorePattern": "^_" }],
       "@typescript-eslint/no-unused-vars": "warn",
```

Biome (2.5.1)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)